content/influxdb/cloud-serverless/write-data/troubleshoot.md

```diff
@@ -68,6 +71,42 @@ If you notice data is missing in your database, do the following:
 
 ## Troubleshoot rejected points
 
-InfluxDB rejects points that fall within the same partition (measurement and day) as existing bucket data and have a different data type for an existing field.
+When writing points from a batch, InfluxDB rejects points that have syntax errors or schema conflicts.
+
+The following HTTP status codes indicate that your batch contains rejected points:
+
+- `201 "Created"`: Some points in the batch are written and some are rejected.
+- `400 "Bad Request"`: The entire batch is rejected.
+
+If some or all points in the batch are rejected, the API response body contains the following properties:
+
+- `code`: the status code description for rejected writes is `"invalid"`.
+- `line`: the line number of the _first_ rejected point in the batch.
+- `message`: a string that contains line-separated error messages, one message for each rejected point in the batch, up to 100 rejected points.
+
+InfluxDB rejects points for the following reasons:
+
+- a line protocol parsing error
+- an invalid timestamp
+- a schema conflict with existing tags or fields in the database
+
+A schema conflict can occur when points that fall within the same partition (default partitioning is measurement and day) as existing bucket data have a different data type for an existing field.
```

Comment:
Points with a tag and field that use the same key will also be rejected. I think these would be considered a "schema" conflict, but I don't know if they'll get a line protocol parsing error or something else. Maybe something to note here.

Reply:
> Points with a tag and field that use the same key will also be rejected. I think these would be considered a "schema conflict," but I don't know if they'll get a line protocol parsing error or something else. Maybe something to note here.

Good catch! @wiedld Do you know when duplicate keys get detected? If not, I'll test post-change and update docs.

Reply:
I believe this should be a schema conflict per line, and therefore the same rules apply (if in partial versus non-partial writes).